src/Microsoft.Azure.WebJobs/ParameterResolverContext.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Class providing context information for a template parameter resolution.
+    /// </summary>
+    public class ParameterResolverContext
```

Comment:
Yes - see how we new up a ParameterResolverContext on each resolve from the call site in BindingTemplate.Bind